### ISSUE TITLE
Folder checker

### ISSUE THREAD

**Issue by master3395** [anderbggo/GooglePhotosMatcher#5]
Hello, if i try and search the main folder with subfolders with images, I get an error.
So my question is.

If I can run this program on the main unzipped folder named Google Foto and then it will check all sub folders for the files and metadata, since the meta data is in the main folder, while images is in subfolders.

**Comment by anderbggo** [comment]
Hi master3395,

Metadata is along with its correspondant media in the same sub-folder. So you just need to run the program in every sub-folder one by one. In the specific case you're describing me (metadata is in the main folder while media in subfolders) you can just move all the media to the main folder and delete subfolders.

I know it will be an easy update to check all the sub-folders recursively, but since it is just a project for personal use, I don't want to spend more time in it (by the moment).

**Comment by master3395** [comment]
Understandable.

I found another project, that has a sub-folder checker working.
So I ran with that one :)
Since I can't download all folders to my computer, I can download 50gig folders, and then run the file and it checks the zip and moves the files into folders or into 1 big folder, sometimes it finds random dates and put images into it.
Like 1902 (I don't have photos from 1902, but I can always just rename the folder :)
But your project helped me on the way :)
![image](https://user-images.githubusercontent.com/4199669/227786934-19e4e267-f12a-4eb9-8d52-ee12dded83ed.png)
![image](https://user-images.githubusercontent.com/4199669/227786955-8295bf15-a790-4e7d-b80c-32c04804102e.png)

https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper
